cmd/http-server/http-server.go

```diff
@@ -28,8 +28,8 @@ func main() {
 	}
 
 	mc := &handlers.MistClient{
-		ApiUrl:          fmt.Sprintf("http://localhost:%d/api2", *mistPort),
-		TriggerCallback: fmt.Sprintf("http://localhost:%d/api/mist/trigger", *port),
+		ApiUrl:          fmt.Sprintf("http://0.0.0.0:%d/api2", *mistPort),
+		TriggerCallback: fmt.Sprintf("http://0.0.0.0:%d/api/mist/trigger", *port),
```

Comment:
Is this right? I thought the communication between catalyst-api and mist was made via localhost. Also, if we try to call 0.0.0.0 it's probably gonna fail, it's only a special value to mean _listening_ on any IP, isn't it?